pkg/operator/ceph/object/objectstore.go

```diff
@@ -289,6 +290,16 @@ func createSimilarPools(context *Context, pools []string, poolSpec model.Pool) e
 			if err != nil {
 				return errors.Errorf("failed to create pool %s for object store %s", name, context.Name)
 			}
+		} else {
+			logger.Infof("pool %s already exists", name)
+			if !isECPool {
+				//detect if the replication is different from the pool details
+				if poolDetails.Size != poolSpec.ReplicatedConfig.Size {
+					if err := ceph.SetPoolReplicatedSizeProperty(context.Context, context.ClusterName, poolDetails.Name, strconv.FormatUint(uint64(poolDetails.Size), 10), poolDetails.RequireSafeReplicaSize); err != nil {
```

Comment:
@leseb Here is an example of where logging would be suppressed from the ceph command. If the logging of ceph commands is info level, the action will be found in the log. If not, we need to ensure we add a `logger.Info()` call everywhere we make an important config change like this.

Reply:
Well I think this will bring overall clarity on logs then if I know nothing about Ceph and CLI and I'm looking from a UI, I'd gladly have a meaningful INFO instead of the cli call...

Reply:
@rajatsingh25aug Please add a log entry before line 298 to indicate that the pool size is changing from the old value to the new value

Reply:
Done!

Reply:
`poolSpec.ReplicatedConfig.Size` is the new size so I think you need to switch the two args to the message

Reply:
Thanks, I also made changes to Line 300 as there also I was using the old variable old pool's size value.